app/frontend/src/internationalization/trans/chefs/hi/hi.json

```diff
@@ -283,98 +278,97 @@
     "about": "के बारे में",
     "loginInfo": "इस फीचर का इस्तेमाल करने के लिए आपको लॉग इन होना होगा।",
     "login": "लॉग इन करें",
-    "401NotAuthorized": "401: अधिकृत नहीं। :",
-    "401ErrorMsg": "आपका खाता ठीक से सेट नहीं है।<br /> कृपया संपर्क करें",
+    "401NotAuthorized": "401: अधिकृत नहीं. :",
+    "401ErrorMsg": "आपका खाता ठीक से सेट नहीं है.<br /> कृपया संपर्क करें",
     "403Forbidden": "403 निषिद्ध। :",
     "403ErrorMsg": "इस पृष्ठ को {idp} प्रमाणीकरण की आवश्यकता है।",
     "401UnAuthorized": "अनधिकृत 401। :",
     "401UnAuthorizedErrMsg": "आपको इस पेज को एक्सेस करने की अनुमति नहीं है।"
   },
   "formDesigner": {
-    "formDesign": "प्रपत्र डिजाइन",
+    "formDesign": "फॉर्म डिजाइन",
     "exportDesign": "निर्यात डिजाइन",
-    "importDesign": "आयात डिजाइन",
+    "importDesign": "डिज़ाइन आयात करें",
     "important": "महत्वपूर्ण",
     "formDesignInfoA": "जब आप इस फॉर्म का निर्माण पूरा कर लें तो <strong>सेव डिज़ाइन</strong> बटन का उपयोग करें।",
-    "formDesignInfoB": "<strong>सबमिट</strong> बटन आपके उपयोगकर्ता को यह फ़ॉर्म सबमिट करने के लिए प्रदान किया गया है और इसे सहेजे जाने के बाद सक्रिय हो जाएगा।",
-    "formLoadErrMsg": "प्रपत्र डिज़ाइन लोड करते समय एक त्रुटि हुई.",
-    "formLoadConsoleErrMsg": "फ़ॉर्म {formId} स्कीमा लोड करने में त्रुटि (संस्करण: {versionId} ड्राफ़्ट: {draftId}): {त्रुटि}",
-    "formSchemaImportErrMsg": "प्रपत्र स्कीमा आयात करते समय एक त्रुटि उत्पन्न हुई।",
+    "formDesignInfoB": "आपके उपयोगकर्ता को यह फॉर्म जमा करने के लिए <strong>SUBMIT</strong> बटन प्रदान किया गया है और इसे सहेजे जाने के बाद सक्रिय किया जाएगा।",
+    "formLoadErrMsg": "प्रपत्र डिज़ाइन लोड करते समय कोई त्रुटि उत्पन्न हुई.",
+    "formLoadConsoleErrMsg": "फॉर्म {formId} स्कीमा लोड करने में त्रुटि (संस्करण: {versionId} ड्राफ्ट: {draftId}): {त्रुटि}",
+    "formSchemaImportErrMsg": "प्रपत्र स्कीमा आयात करते समय एक त्रुटि उत्पन्न हुई.",
     "formSchemaImportConsoleErrMsg": "प्रपत्र स्कीमा आयात करने में त्रुटि: {त्रुटि}",
-    "formDesignSaveErrMsg": "इस प्रपत्र डिज़ाइन को सहेजने का प्रयास करते समय एक त्रुटि हुई. यदि आपको बाद में पुनः प्रयास करने के लिए ताज़ा करने या छोड़ने की आवश्यकता है, तो आप बाद में सहेजने के लिए पृष्ठ पर मौजूदा डिज़ाइन निर्यात कर सकते हैं।",
-    "formDesignSaveConsoleErrMsg": "फ़ॉर्म को अपडेट करने या बनाने में त्रुटि (FormID: {formId}, versionId: {versionId},raftId: {draftId}) त्रुटि: {error}",
+    "formDesignSaveErrMsg": "इस प्रपत्र डिज़ाइन को सहेजने का प्रयास करते समय एक त्रुटि उत्पन्न हुई। यदि आपको बाद में पुनः प्रयास करने के लिए रीफ़्रेश करने या छोड़ने की आवश्यकता है, तो आप बाद के लिए सहेजने के लिए पृष्ठ पर मौजूदा डिज़ाइन को निर्यात कर सकते हैं।",
+    "formDesignSaveConsoleErrMsg": "फ़ॉर्म को अपडेट करने या बनाने में त्रुटि (फ़ॉर्मआईडी: {formId}, संस्करणआईडी: {versionId}, ड्राफ्टआईडी: {draftId}) त्रुटि: {त्रुटि}",
```

Comment:
The last parameter `{error}` for formDesignSaveConsoleErrMsg looks like it was accidentally translated here.